std_msgs/String get_marker_req
---
environment_marker get_marker_res
bool is_marker_exist
